lib/rules/curly.js

```diff
@@ -178,5 +237,8 @@ module.exports = function(context) {
 module.exports.schema = [
     {
         "enum": ["all", "multi", "multi-line", "multi-or-nest"]
+    },
+    {
+        "enum": ["consistent"]
```

Comment:
I was hoping that could be done through schema. @btmills some help with this please.